### ISSUE TITLE
use colorblind-friendly color scheme

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#68]
In our current scheme, the blue color does not work against the dark mode background:

https://www.colorhexa.com/173d63

**Comment by marisademeglio** [comment]
fixed in https://github.com/daisy/pipeline-ui/commit/cf8036ec0146c6b236a25d5551b1c00126bf645c
